### ISSUE TITLE
Add support for service-2-service communicaion

### ISSUE THREAD

**Issue by michaelwittig** [cfn-modules/fargate-service#30]

**Comment by tjtaill** [comment]
Yeah love your modules but I have a use case to work with service registry https://aws.amazon.com/premiumsupport/knowledge-center/cloudformation-ecs-service-discovery/

Would be great if this module supported it
